content/general/concepts/file-formats/file-formats.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'File formats'
+Description: 'File formats determine how information or data, such as text and images, are created, stored, and read.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Files'
+CatalogContent:
+  - 'paths/code-foundations'
```

Comment:
Hmmm I'm having a hard time finding a suitable `learn-` link for this one 😂

Reply:
😆 what do you think about `learn-the-command-line`?

Reply:
Works for me 😃

Reply:
```suggestion
  - 'learn-the-command-line'
  - 'paths/code-foundations'
```